cl/phase1/network/gossip_manager.go

```diff
@@ -101,54 +105,16 @@ func (g *GossipManager) onRecv(ctx context.Context, data *sentinel.GossipData, l
 	// then attempts to deserialize the received data into it.
 	// If the deserialization fails, an error is logged and the loop returns to the next iteration.
 	// If the deserialization is successful, the object is set to the deserialized value and the loop returns to the next iteration.
-	var object ssz.Unmarshaler
 	switch data.Name {
 	case gossip.TopicNameBeaconBlock:
-		object = cltypes.NewSignedBeaconBlock(g.beaconConfig)
-		if err := object.DecodeSSZ(data.Data, int(version)); err != nil {
-			g.sentinel.BanPeer(ctx, data.Peer)
-			l["at"] = "decoding block"
-			return err
-		}
-		block := object.(*cltypes.SignedBeaconBlock)
-		l["slot"] = block.Block.Slot
-		currentSlotByTime := utils.GetCurrentSlot(g.genesisConfig.GenesisTime, g.beaconConfig.SecondsPerSlot)
-		maxGossipSlotThreshold := uint64(4)
-		// Skip if slot is too far behind.
-		if block.Block.Slot+maxGossipSlotThreshold < currentSlotByTime {
-			return nil
-		}
-		if block.Block.Slot == currentSlotByTime {
-			if _, err := g.sentinel.PublishGossip(ctx, data); err != nil {
-				log.Debug("failed publish gossip", "err", err)
-			}
-		}
-
-		count, err := g.sentinel.GetPeers(ctx, &sentinel.EmptyMessage{})
-		if err != nil {
-			l["at"] = "sentinel peer count"
-			return err
-		}
-
-		log.Debug("Received block via gossip",
-			"peers", count.Active,
-			"slot", block.Block.Slot,
-		)
-		g.gossipSource.InsertBlock(ctx, &peers.PeeredObject[*cltypes.SignedBeaconBlock]{Data: block, Peer: data.Peer.Pid})
-	case gossip.TopicNameLightClientFinalityUpdate:
```

Comment:
so we actually don't need to handle these two cases? i mean TopicNameLightClientFinalityUpdate and TopicNameLightClientOptimisticUpdate

Reply:
Ye we dont really do anything and do not even listen to them yet.